app/controllers/standup_meetings_controller.rb

```diff
@@ -1,7 +1,23 @@
 class StandupMeetingsController < ApplicationController
+  def index
+    @meeting_date = params[:date].nil? ? Date.current : Date.parse(params[:date])
```

Comment:
I am debating on whether this should be a required param and the routes pass it in (in like 95% of the cases they will).

Reply:
I think it probably makes sense to keep, just in case?